python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -407,6 +427,32 @@ def test_torch_tensor_nccl_nested_dynamic(ray_start_regular):
     compiled_dag.teardown()
 
 
+@pytest.mark.parametrize("ray_start_regular", [{"num_cpus": 4}], indirect=True)
```

Comment:
can we add some comments to explain the test?

Reply:
It's pretty self explanatory from the error message.
Also I simplified the test further so that it is more straightforward.